cpp/src/rolling/grouped_rolling.cu

```diff
@@ -93,6 +93,19 @@ std::unique_ptr<column> grouped_rolling_window(table_view const& group_keys,
 
 namespace detail {
 
+/// Checks if it is possible to optimize fully UNBOUNDED window function.
+bool can_optimize_unbounded_window(bool unbounded_preceding,
```

Comment:
It would be better to use a header for a repeated declaration from another source file.